src/stores/BalanceStore.ts

```diff
@@ -31,6 +43,19 @@ export class BalanceStore {
     },
   };
 
+  myMarketBalanceList = {
```

Comment:
Лучше убрать 0 элемент и сделать пустым по умолчанию. Еще вместо объекта можно использовать Map, но это так, в целом без разницы

```suggestion
interface ContractBalance = {
       locked: {
         base: BN;
         quote: BN;
       };
       liquid: {
         base: BN;
         quote: BN;
       };
     };

  myMarketBalanceList: Record<string,  ContractBalance> = {}
```